### ISSUE TITLE
GroupEntitlements API does not return members if they are Azure AD groups

### ISSUE THREAD

**Issue by debashis1982** [MicrosoftDocs/vsts-rest-api-specs#193]
If a VSTS group created inside a project has users as well as Azure AD groups as its members, the GroupEntitlements API returns only users and not AD group members. API URL - 

GET https://vsaex.dev.azure.com/{organization}/_apis/GroupEntitlements/{groupId}/members?api-version=5.0-preview.1
should it not return AD Groups also?

in contrast, the team API seems to return both users and AD groups. API URL - 
GET https://dev.azure.com/{organization}/_apis/projects/{projectId}/teams/{teamId}/members?api-version=5.0

Should we not expect both APIs to behave identically in terms of the responses they return?

**Comment by richardzaat** [comment]
I completely agree as today I was surprised to not see the groups. after all, they are a "member" of the other group.
